src/bit-u.md

```diff
@@ -530,11 +532,11 @@ a.quote-link:hover{
                   <span class='event-group'></span>
                 </p>
               </div>
-              <p class='event-body'></p>
-              <a class='event-url button button-grey full-width' >Register for event</a>
+              <p class='event-body' style="word-break: break-word"></p>
```

Comment:
Normally, we'd want this in the stylesheet not inline. But Academy styles will probably be redone later, so its not a problem to do a small weird thing now.